scripts/spack/packages/axom/package.py

```diff
@@ -395,6 +405,14 @@ def initconfig_mpi_entries(self):
         else:
             entries.append(cmake_cache_option("ENABLE_MPI", False))
 
+        # Replace /usr/bin/srun path with srun flux wrapper path on TOSS 4
```

Comment:
@white238 
I added a snippet here to target the flux wrapper for srun, instead of slurm srun for toss 4 machines. This is a follow-up addition to PR #1124.